sdk/src/signer/keypair.rs

```diff
@@ -60,8 +59,8 @@ impl Keypair {
     }
 
     /// Gets this `Keypair`'s SecretKey
-    pub fn secret(&self) -> &ed25519_dalek::SecretKey {
-        &self.0.secret
+    pub fn secret(&self) -> ed25519_dalek::SecretKey {
```

Comment:
I think this is a backward incompatible change, because `ed25519_dalek::SecretKey` is changing its version from `1.0.1` to `2.x`.
Maybe it is OK here, but a similar change in `PubKey` for the `borsh` derivation from `0.9` to `0.10` created a lot of issues.

Reply:
Let me know if you want to discuss the transition.
I was helping Leo with the borsh compatibility issue.